install-dynamodb-local.sh

```diff
@@ -0,0 +1,23 @@
+
+#!/bin/bash
+
+
+echo "Installing Java"
+brew install openjdk
+
+echo "java is installed sucessfully.... :) "
+echo " "
+echo " "
+
+mkdir ./dynamolocal && cd ./dynamolocal
+
+
+echo "Downloading the DynamoDB tar file" 
+wget http://dynamodb-local.s3-website-us-west-2.amazonaws.com/dynamodb_local_latest.tar.gz
+tar xzf dynamodb_local_latest.tar.gz
+rm -f dynamodb_local_latest.tar.gz
+
+echo " "
+echo "Dynamodb-local is installed and ready for use.... :) "
```

Comment:
Would be useful to add the command to start, how to ping the local installation, and a link to the docs. 

```suggestion
echo "Dynamodb-local is installed and ready for use.... :) 

To run DynamoDB locally: java -Djava.library.path=./DynamoDBLocal_lib -jar DynamoDBLocal.jar -sharedDb

To point AWS CLI to Local DynamoDB: aws dynamodb list-tables --endpoint-url http://localhost:8000

For more details, see the documentation: https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.DownloadingAndRunning.html#DynamoDBLocal.DownloadingAndRunning.title

"
```